contributing.md

```diff
@@ -46,3 +46,121 @@ If you are proposing a feature:
 * Remember that this is a volunteer-driven project, and that contributions
   are welcome :)
 
+#### Add Translations
+
+All translations are handled by `transifex <https://app.transifex.com/nephila/>`_.
+
+If you want to contribute on translations, request an access to our `transifex` team and we'll add you
+as soon as possible.
+
+### Get Started!
+
+Ready to contribute? Refer to the project `Get Started!` section of `CONTRIBUTING.rst`
+to set up your environment for local development. 
+
+We use, for most projects, this set of packages:
+
+- `bumpversion`_
```

Comment:
links must be updated to use markdown syntax, original document is in REST format

Reply:
@yakky Should be fixed now